input/core/src/commonMain/kotlin/majestic/editor/toolbar/ToolBarHost.kt

```diff
@@ -28,13 +28,15 @@ import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import majestic.editor.body.chunkUI.DropDownColors
 
-class ToolBarTabColors(
+class EditorColors(
     val text: ToolButtonColor = ToolButtonColor(Color.White, Color.Gray),
     val underline: ToolButtonColor = ToolButtonColor(Color(0xFF0061FF), Color.Gray),
     val background: Color = Color.Transparent,
     val foreground: Color = Color.White,
-    val brush: Color = Color.White
+    val brush: Color = Color.White,
+    val dropDownColors: DropDownColors = DropDownColors()
```

Comment:
instead of `dropDownColors`, you can just call it `dropDown`. Otherwise the call site will endup reading `colors.dropDownColors` instead of just `colors.dropDown`